sqlglot/dialects/snowflake.py

```diff
@@ -1388,6 +1398,7 @@ class Generator(generator.Generator):
 
         TRANSFORMS = {
             **generator.Generator.TRANSFORMS,
+            exp.AddMonths: lambda self, e: self.func("ADD_MONTHS", e.this, e.expression),
```

Comment:
So if you replace `exp.true()` with `True` in the constructor, you won't need this entry because booleans are excluded when generating function arguments. I think this happens in `format_args` if you want to see where it happens.